presto-docs/src/main/sphinx/security/server.rst

```diff
@@ -0,0 +1,260 @@
+==========================================
+Presto Coordinator Kerberos Authentication
+==========================================
+
+The Presto coordinator can be configured to enable Kerberos authentication over
+HTTPS for clients, such as the :doc:`Presto CLI </security/cli>`, or the
+JDBC and ODBC drivers.
+
+.. warning::
```

Comment:
Let's swap the order of this and the second paragraph. I think it makes more sense after reading that paragraph.